### PR TITLE
Neurotoxin and Beepsky Smash Tweaks

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Generally speaking, most chems have moved away from being instantaneous stuns/weakens/paralysis unless they're traitor in origin (this has been the treatment prior to the introduction of Goofchem ala Chloral Hydrate...and has continued under it). 

As such, there's two major standouts that are incredibly easy to mass produce that are very similar to old chloral hydrate, in behavior.  Those two standouts are Neurotoxin and Beepsky Smash.

:cl: Fox McCloud
tweak: Neurotoxin and Beepsky smash no longer instantly stun; there's a delay to them before they kick in
/:cl:

### CODE REVIEW

**Comment by PKPenguin321** [comment]
lame

**Comment by KazeEspada** [comment]
:+1:

**Comment by Zilenan91** [comment]
No. :-1:

**Comment by newfren** [comment]
I'm like 90% sure that Beepsky smash doesn't stun it just makes you drop shit.

**Comment by Fox-McCloud** [comment]
>I'm like 90% sure that Beepsky smash doesn't stun it just makes you drop shit.

That used to be the case, back when it only stunned for 1 cycle, but that's no longer the case, and it's a full blown stun, since the duration is `2` now.

**Comment by Eaglendia** [comment]
:-1: If things like this are legitimately too powerful to be in the game, they'd be meta, or even commonly utilized or looked to as tools for antagonizing. Consider that further sterilizing on-station tools limits options for both antags and nontags, without strengthening anything except metagaming potential.

**Comment by sirnat** [comment]
Literally nerfing neurotoxin only because it's been used recently, and I think I've been the only one who really uses it to its potential. 

It's suppossed to be a quick non-lethal stun that only lasts a small amount of time, plus you can still yell your location clearly over radio!

I feel like mekhi when the RD armor got nerfed.

**Comment by Fox-McCloud** [comment]
@sirnat 

Only, you can't scream for help. Anyone who can make beepsky smash can also make perfluoradecalin---a single syringe with 10 units smash/neurotoxin and 5 units perflu is enough to keep you stunned for over 50 seconds and silenced for over 100 ....and chemistry/science can easily mass produce both, especially thanks to the abandoned bar in maintenance.

**Comment by PKPenguin321** [comment]
then make it metabolize faster to effectively shorten the stun or something. with this change, why would i EVER use these chems over, say, basic morphine?

**Comment by Zilenan91** [comment]
This sounds more like i ded than anything else. It's fine that someone can do something powerful every once in a while, but it isn't a problem because nobody spams this.

**Comment by Fox-McCloud** [comment]
@PKPenguin321 

Because they're going to get your target drunk as hell in the process, not to mention knock out faster than Morphine  because morphine takes 24 cycles to knock out (as opposed to the 15 for these).

**Comment by PKPenguin321** [comment]
who cares if they're drunk? unless you somehow get like 500u into their system that will only be a joke of a downside at best.

with this much of a delay they can still run a fair distance and call for help. it's effectively the same thing as morphine, only harder to get. :-1:

**Comment by Zilenan91** [comment]
Also god forbid anybody does anything but make a stunprod and buy a holoparasite to murderbone for their traitor round

**Comment by Fox-McCloud** [comment]
And both pale in comparison to the still retardedly powerful chloral hydrate, which is easier to acquire than any of the reagents mentioned in this conversation. I really don't see how that's a valid point.

**Comment by Cheridan** [comment]
I don't really see this being abused much. With old chloral it was a huge meta thing, every CMO had a hypo full of it, every chemist rushed for the syringe gun. The big thing about this is that the bartender who can make these chems doesn't typically have a very good application method for them.

**Comment by Fox-McCloud** [comment]
@Cheridan 

It's not the bartender that's really the problem, TBH--it's the fact that there's a full dispenser and booze-o-mat in maintenance that everyone and their brother can grab, then make as much of this as they desire.

**Comment by Zilenan91** [comment]
The problem isn't that they **CAN** do it, the problem is if they **WILL**

Nobody is doing it so it isn't a problem.

**Comment by SweaterKittens** [comment]
Yeah, I'm with Fox on this. Not only for game balance, but it's silly that alcohol makes a stronger stun than a dedicated chemistry lab. It doesn't fit with the way the game is currently. No one has abused it currently, but it doesn't take a genius to see that it's easily abusable, doesn't make sense with the current chem situation, and we lose nothing by being rid of it.

**Comment by AdamElTablawy** [comment]
Hi.

I abuse the shit out of neurotoxin and entirely agree it's overpowered as fuck.

As an instant stun easily mixable with pleufloradecalin put in a mining minihypo is almost exactly what oldchloral was minus they're not unconcious, so anyone looking won't try to get them up.

I've killed all of security about six times now with pleuflora neurotoxin in a miner minihypo or the regular hypo.

It's strong enough to btfo anything short of a xeno queen or praetorian, and if you use it in a dropper and aim it at their eyes, they're still instantly stunned and you can chainstun the shit out of them. Granted they have to be bad not to tailsweep you but it's still obnoxious.

A neurotoxin gas grenade is grossly overpowered. I used it on a wizard with a construct army and mjolnir-suicide neurotoxin bomb, he was down, unmoving and unable to speak, screen flipping the fuck out due to liquor effects, for fifteen fucking minutes, until he got lasered to death.

I gassed about 7 revs in arrivals with one-they were down for so long it took 9 minutes for sec to arrive and cuff them all, and even after they got implanted 8 minutes later some were still stunned and some were shaking, tripping the fuck out and passing out.

It's hideously, hideously unbalanced-you can knock someone out of the round for a ridiculous amount of time with just one neurotox pleuflora grenade, and in a syringe or hypo it's an instant mutestun that's more than enough time to kill them, and if using a hypo chances are nobody will see it happening until you've stunned everyone in the room.

I started the neurotoxin trend, and let me tell you, a chemical that instantly grants you complete superiority over your opponents that even an assistant can gain access to without even entering medbay, that can keep people down for 1/4 of a 40 minute round muted and stunned, that permanently fucks someone's vision and movement even after the stun is gone, is nastily, nastily overpowered.

You can kill the shit out of anyone or anything not wearing a hardsuit or biosuit with this, and a minihypospray will go right through their shit and stun them instantly. I remember killing three ops by hiding in a locker, running out and jabbing them all with a minihypo of neurotoxin as a fucking miner medic and beating them to death with floor tiles.

I am fully and completely in support of either making the recipie harder or nerfing it as you did-i've doled out neurotoxin and been on the recieving end of it, and it's fun for nobody. It's an instant win if a single blast of it connects.

A spraybottle of neurotoxin and pleufloradecalin has killed entire gangs singlehandedly before. It's the most absurd shit.

100% in support of a nerf-it's not 'hugbox', it's not 'wahh this is i ded pls nerf', this is a legitimately overpowered chemical that can't be planned for unless you're el powergamer extremeo eating charcoal and antihol, because if you're within three tiles of someone, you can get mutestunned by a bottle of neurodecalin; and if you're in eyesight of someone, literally any job at all, they likely have an impro shotgun with a shotgun dart of neurotoxin, a damp rag soaked in neurotoxin, or a syringe gun of neurotoxin, and one shot of any of those means that you're fucking dead, because when you're stunned and muted for that long there's no way out of it and chances are unless they're retarded and do it in a public hallway the maybe one or two people that also see are also instantly getting stunned and muted. It's like a ranged parasting, it's like a medium range taser or an energy crossbow that also instantly mutes.

It's unfair, and this is coming from the guy that started the whole trend and who abuses the shit out of it. I support a nerf.

**Comment by PKPenguin321** [comment]
>the guy that started the whole trend

kek
every spaceman worth their salt has "invented" the use of  this at some point

>And morphine/neurotoxin/beepsky smash pale in comparison to the still retardedly powerful chloral hydrate, which is easier to acquire than any of the reagents mentioned in this conversation.

morphine is literally in medivends roundstart, at least chloral requires chemistry access

**Comment by Zilenan91** [comment]
Actually yeah this nerf makes it pointless to use it over morphine

**Comment by AdamElTablawy** [comment]
I started using it again and shouted how good it was in OOC and then everyone went 'oh right neurotoxin is good' and then people started using it again.

**Comment by PKPenguin321** [comment]
Give it a round or two to die down again, then.

**Comment by killerx09** [comment]
We could just make it more complicated to make.

**Comment by AlexanderUlanH** [comment]
>Rebalancing unintuitively strong and relatively obscure things that only exist because they were missed when things were redesigned only helps metagamers

How exactly? That kind of stuff is what metagamers thrive on. When the tools a metagamer relies on are the things a roleplayer will intuitively try to use, that's good game balance.
A drink shouldn't be the powergamer CMO's poison of choice. It should be a drink. The CMO looking to poison people or make gas grenades should go be making poisons.

**Comment by Lumipharon** [comment]
What even is this thread?
Chemists can make shit that will KILL you with a syringe, so can botanists.

If shit like mining hypos can be refilled, then that's a balance issue with them, but the CMO is the only person that has a repeatable, reliable delivery method, short of someone making an assload of shotgun darts/has an oddy, both which require significant time investments.

If shit like perflu is overpowered (which it honestly is, considering how much more difficult actual mute toxin is to make), than nerf that.

Saying people that get access to chemistry shit, and bartending shit, and a good delivery method can abuse a chem when combined with another unbalanced chem... Hm.

Also chloral hasn't been an instant stun since far, far before goofchem ever got ported.

**Comment by AdamElTablawy** [comment]
It's fairly easy to get a good delivery method for this gg stunchem.

Go to the bartender, ask for pan galactic gargle blaster. Chances are he's bored and makes it for you. Go to med, hack a medvendor or nag a doctor or something, grab one or two bottles of morphine.

Now you have options.

You can rig up a ghetto neurogas grenade by climbing into the kitchen and grinding some misc shit for sugar, or ask chemistry for smoke powder. That's the most difficult delivery method.

Spraybottle? With the janitor, one by chem, one in med storage, one in viro. And a ton in botany or the garden, meaning a nice 3-5 tile range of hard to see, hard to dodge, hard to anticipate gg chem that floors you immediately and leaves your screen spazzing out for the next few minutes with the drunk effect.

Damp rag? You can get that shit anywhere.

Syringe gun? Break into r&d, print a rapid syringe gun. Or just ask the guy doing r&d for one, they love to give guns out to everyone.

The list goes on with beakers, feeding, minihypos, droppers to the eye... An assistant can make 100u of this shit in 5 minutes where even the nastiest deathmixes require chemistry access, and time, and skill in knowing which chem works best with which.

The problem is only partially the absurd stun, and the stupid decay, where you can puff someone once or twice and keep them down for a full minute unless they're in a hardsuit, hopped up on antihol, charcoal, and meth all combined, or a ling. Not even adrenals, jaunt, telearmor, any sort of armor, can block this shit in any way-the only way to be completely secure is to be in a biosuit or hardsuit with internals turned on, or to be chugging antihol, charcoal, and meth all at once. 

Now, i'm not saying that shit like this isn't available elsewhere. Stunprods are easy to make and also an instant stun.

But unlike neurotox there's only one delivery method, the method is melee ranged and obvious, and it's never overlooked by sec.

With a spraybottle of neurotox-which you can get shitloads of from the garden or botany or janitor or cargo- the only warning you have is 'pfft pfft' and you're on the ground, stunned, slurring, and tripping the fuck out.

What makes it worse than this, and it's insane every single job style accessibility, is that it's legitimately superior to any other chem you can make in chemistry, and even some of the deathmixes.

With a syringe shot of say, formaldehyde and clf3, the guy has a chance to figjt back before dead. Even a murder cocktail of traitor poisons like initro and coiine mixed together give you precious seconds to fight back and scream.

With neuro?

3-5 minutes of prep, unsuspicous to any job, pfft pfft, they're on the floor stunned for a good thirty seconds to a minute, long enough for you to spray or chase off any witnesses and kill the whole group at lesiure.

Most murder cocktails for even the most skilled chemists require syringe gun shots, patch applications, take eight or ten minutes of prep or mixing, access to chemistry and a chem dispenser uninterrupted, and are obvious in their effects when you poke someone and they shit their own heart out and catch fire. 

With neuro, one puff and you're down for the count, no chem access required whatsoever. And it's even worse in concentrations over 10u-slap a 50u patch on them or toss a neuro smoke grenade in, they're locked down for about fifteen minutes.

Basically makes the wizard gem hardsuit a required buy-if you don't take it, you get suicide neurotoxin bombed and you're helpless for fifteen minutes and can only annoy people as they laser you.

It's like adding a three to five tile ranged stunprod, with upwards of 50 charges in the prod, each one stunning for ten or fifteen seconds, with almost no cooldown to hits, and being craftable with a pipe fitting and an igniter and then arguing that it isn't abused yet so it's not a problem. 

But it is, because it's obnoxious and overpowered and makes no sense that any job with five minutes can instantly outdo a traitor chemist at their one iob-making poisons.

**Comment by PKPenguin321** [comment]
>it's fairly simple to get a transfer method
>40 paragraphs of the """simple""" process

nigga

**Comment by AdamElTablawy** [comment]
Okay apparently you're illiterate. Let me try again in simpler words.

 be assistant.

go to bar.

get pan galactic.

Go to med. 

Loot one of the 5 suicided doctors or geneticists for an id.

Get morphine.

Get one of the 3 spraybottles from medbay or the 5 scattered around the station or the infinite in the biogen.

Kill everyone.

**Comment by Zilenan91** [comment]
Or don't because this relies on you having a ridiculous amount of morphine to make, so at most you're going to kill only 3 or 4 people, if even that.

**Comment by PKPenguin321** [comment]
>get sprayed by adam's meme juice
>oh no 5u of stuns!
>call for help because it doesnt even knock you out
>5u wears off almost immediately and you walk away
>adam gets raped

still not feeling it

**Comment by Lumipharon** [comment]
Ok I forgot a couple of relatively recent changes that effect this.

1) Neuro and beepsky now cause actual alcohol effects (they didn't used to be alcoholic, or atleast I never noticed it)

2) That change that made chem transfer methods more varied and specific. (You used to not be able to use neuro on people with touch application and shit, which meant no spraying)

That said, the chem itself is still not an issue TBH. Your "simple method" involves getting a specific drink from the bartender/breaking into there, getting medbay access somehow and looting all the morphine/syringe gun/syringes/spray without anyone thinking you're a suspect fuck, then you've got a 3 tile range (spray) or 1 shot (syringe gun) weapon to use, and that's still without a method to mute people, as perflu DOES require chem access. And then of course a hardsuit/biosuit/mech/borg/standing outside of spray range all hard counter this shit.

A syringe gun filled with neuro is still a weaker e-bow.
A spray, although it can be spammed, has THREE tile range.
No other application method is notably superior to other takedowns either.

This shit is strong, but it requires some level of effort to make, and is far from OP.

**Comment by AdamElTablawy** [comment]
It takes one or two sprays from a spraybottle of neurotoxin to down someone for around thirty seconds. This is 10u, at most 20u, of neuro used.

One bottle of morphine and 30u of pan galactic will get you 60u neurotoxin.

So either 6 people for one bottle or if you want to risk it and use 5u, which will only down someone for about 8-10 seconds, that's 12 people dead with one fucking bottle of morphine and a spraybottle, assuming your puffs connect and you don't get cucked by a borg.

A medvendor will have 4 bottles of morphine, a total of 120u.

120u morphine and 120u pan galactic, easily gotten within five minutes of roundstart as an assistant by climbing into the bar and hacking the medvendors.

That's 280u of neurotoxin, when 5u or 10u is all that's needed to beat someone to death.

And spraybottles perfectly capable of administering that much over a 3 or so tile distance with a subtle inhand sprite and capable of carrying 250u are everywhere.

That is, assuming you get in melee range of someone, or a one or two tile distance of them, anywhere between 15 to 28 easy kills as they're on the floor stunned long enough for you to kill them. 

And assuming that because you're doing this you're also a traitor you likely have an emp flashlight and adrenals, meaning you can mute the headsets of those you stun, no pleuflora needed, and stun resist for if sec does make it past the neurostorm.

Hell even if you're not a traitor you're either a group antag surrounded by loads more group antags who will happily kill the people you stunned, or a validhunter perfectly equipped to kill anyone or anything short of a nukeop or changeling-even shadowlings will get cucked. Even wizards, as several can attest, one puff of neuro is enough to chainneuro them, block their jaunt and beat them to death.

So 3-4? Not going to happen. A skilled and stealthy traitor with one bottle using it to maximum effectiveness can kill 25 people- even someome with shitty aim and sub par combat skills can ambush and kill all of security within 5 minutes of roundstart regardless of access or restrictions, because puff puff stunned dead. Sec storming you? Neuro gas grenade, they're fucked for upwards of fifteen minutes.

Way more than 3-4. On a regular basis, 10-12, from one trip to the bar and a medivendor.

**Comment by AdamElTablawy** [comment]
It requires about as much effort as a stunprod and is less suspicious than having one. Nobody notices nor cares about the assistant in med storage or taking the spraybottle from beside chem. 

Nobody cares about the engi taking pest spray from the arrivals garden or botany until 15 minutes later when they're visiting space without a suit, courtesy of the engineer.

Does it take effort? Yeah, so do stunprods, spears, pneu cannons, ect.

Is it hard to make or get? No, just like stunprods, spears, pneu cannons, ect.

And unlike the above, almost all delivery methods fit in your pocket. Puff puff, take headset/mute with flashlight/just hit them a bunch, dead.

Absolutely dead, gg, ur fucked.

Is it hard countered by some shit? Yeah, it is.

Things can be hard countered and still be OP.

Old GBS or beesease + anxiety or fungal tuberculosis is hard countered by wearing a biosuit or hardsuit and having internals on: You still ended up dead usually.

You might argue that with the virus there's no direct assailant unlike neuro that you can instantly stop, and you'd be right.

But on the other hand, those viruses were very obvious, people would be wearing biosuits and hardsuits, and STILL get killed.

With neuro, unless someone is being blatantly obvious about it, people won't rush for the hard counter biosuit/hardsuit shit because they won't expect it and that hard counter slows them down. 

The neuro guy picks and chooses his fights, and as a result he can kill almost anyone not constantly in a hardsuit or biosuit-meaning, one atmos tech, 1-2 engineers, the CE, and the virologist will generally be immune. That's 5 people out of a regular 60 pop.

5 people.

And for them, well, chances are since he's an antag with neuro if he's not an idiot he'll have some method of killing them, say, a revolver.

Or just ignore them entirely, since unless he's openly murderboning he can choose who he fights and just depopulate entire departments slowly with a spraybottle spritz, a headset removal and a screwdriver to the eyes.

On it's own, neurotoxin is alarmingly formidable in the hands of an open murderboner killing everyone with it and fighting 3v1.

In a 1v1, neurotoxin is superior to almost anything else-unlike anything like a taser or syringe gun it's a subtle inhand sprite that fits in your pocket, works instantly, and is fully capable of killing lone or couples of people roaming around, especially if they do not expect it. And unlike a stunprod or anything else, a bottle or damp rag or smashing an alcohol bottle of it over their head or even a falsely labeled patch is far subtler and less blatantly dangerous than a tazer or prod or anything else until it is applied.

Can someone with neuro be stopped? Yes. Unless they've taken some serious prep or are a ling or adrenal packing traitor, one taser will have them dead to rights.

Can they be stopped easily, especially if they're not openly murderboning but just killing lone or couples of people in isolated spots? No, not at all. And for something that's so easy to get at roundstart, that is worrying.

**Comment by Zilenan91** [comment]
You're writing an unnecessarily large amount about this

**Comment by Cheridan** [comment]
leaning towards merging this as adam supports it and he obviously has a doctorate on the subject

**Comment by AdamElTablawy** [comment]
5u doesn't wear off immediately, as I have been screaming my head off. 

It fucking sticks. 

One or two puffs of neuro will put you down for the count for ten to fifteen seconds. A stunbaton stuns you for I believe 8.

And by puff, I mean the 5u of neuro coming from a spraybottle. One syringe of neuro will floor you for thirty or so seconds. Spritz someone three times with a spraybottle for the same effect.

Snark all you want. Go try it out on someone, come back and tell me it's perfectly balanced and you couldn't use it to kill a shitload of people with ease, or chainstun a wizard. Toss a neuro gas grenade and come back 15 minutes later and tell me if you're still stunned.

And god forbid you pair it with something like the fucking 90 mute abilities antags have, like emp talismans, emp flashlights, mutestings, glare, resonant shrieks, or even stunning you and immediately sawing you to death while your 'call for help' becomes ';hel-GLORF'

**Comment by Zilenan91** [comment]
I mean how dare any traitors do anything but build a stunprod and buy a holoparasite for their traitor rounds. How dare we have anything powerful and rarely used in our codebase.

**Comment by AdamElTablawy** [comment]
I don't have a doctorate in neurotoxin, only a master's.

Also telling random strangers who disagree with me on something that they're wrong is my fetish.

The faster you merge this the faster I'll fuck off. Also all my points still stand, so.

**Comment by PKPenguin321** [comment]
>5u doesn't wear off immediately, as I have been screaming my head off.

It fucking sticks.

One or two puffs of neuro will put you down for the count for ten to fifteen seconds. A stunbaton stuns you for I believe 8.

then make it metabolize faster like i already mentioned further up ?????

**Comment by AdamElTablawy** [comment]
Or, because it's a fucking bar drink do exactly what this pr is doing and give barneuro's effects to space drug chem heater Neuro so that horrible deathchems are once again gated behind chem.

**Comment by PKPenguin321** [comment]
>exactly what this pr is doing and give barneuro's effects to space drug chem heater Neuro

that's not what this PR is doing though

**Comment by AdamElTablawy** [comment]
I said AND

As in in addition to this pr's change

**Comment by Lumipharon** [comment]
Yeah, no.

It requires access to multiple departments/areas of maint to prepare, where as a prod require all of 5 seconds in tool storage/science/engineering/etc to make. A pneumatic cannon takes even more prep, is time consuming to load, and after that, **CAN INSTANTLY KILL PEOPLE IN ONE FUCKING HIT**.

If people are ok with pneumatic cannons that literally anyone with a wrench can make, then they can damn well be ok with a short range stun that requires far more effort and actual access to make.

**Comment by KazeEspada** [comment]
Pneumatic Cannons needs a pipe. Good luck getting into atmos! /me closes the blast doors.

**Comment by Zilenan91** [comment]
Just wrench out some air pipes from maint, there's a bunch above prime tool storage

**Comment by PKPenguin321** [comment]
^

**Comment by Anonmare** [comment]
"Salt: The PR" tbh fampai

**Comment by Zilenan91** [comment]
Yeah I made pneumatic cannons all the time as an assistant. You get fucked if you miss though because it stuns you.

**Comment by PKPenguin321** [comment]
Not if you set it to 2

**Comment by Zilenan91** [comment]
If you set it on 2 not all shards embed afaik

**Comment by Lumipharon** [comment]
>Pneumatic Cannons needs a pipe. Good luck getting into atmos! /me closes the blast doors.
>literally every room/corridor/maint corridor is filled with pipes waiting to be wrenched

Literally wat.

Also pneumatic will still instacrit when on 2.

**Comment by Incomptinence** [comment]
These are the chemicals you deserve. 

Space men are spawned to die.

**Comment by Fox-McCloud** [comment]
@Lumipharon 

Unless they're throwing deathnettles, there's nothing on station that can instacrit you with the cannon unless you already had some decent damage to begin with, and armor is always a factor.

**Comment by Lumipharon** [comment]
I have been insta critted more than once, as a sec officer with armour, by a pneumatic cannon filled with glass shards.
Do you not remember that fuckhuge thread/pr raging about how OP it was that any random could make a 1 hit crit weapon out of random scrap, vs the shitty improv shotgun that requires R&D work?

**If** it's been severely nerfed since then, well ok, but I'm not aware of that.

**Comment by Zilenan91** [comment]
It got nerfed from 10 shards to 7 but it's still good

**Comment by Fox-McCloud** [comment]
@Lumipharon  What @Zilenan91 said; shards do 10 damage, and the various settings of the pneumatic have zero impact on how much damage they do (they only alter range and speed...both of which are basically pointless when you can't shoot past the screen range and speed has negligible impacts).

Shards are the best thing you can access for them outside of hatchets and deathnettles---and hatchets are VERY limited.

**Comment by Zilenan91** [comment]
Hatchets are worse for it overall, the majority of the damage that the pneumatic does is bleeding.

**Comment by PKPenguin321** [comment]
embedding itself does loadsa damage which does most of the legwork, but the bleeding tends to finish dudes off

this is way off topic though
:-1: to this PR

**Comment by demonfiren** [comment]
i ded tag when?
Cos this is utter shit.

**Comment by duncathan** [comment]
this isn't even controversial; it's pretty clear almost nobody wants this.
